src/data/ha-state-history-data.html

```diff
@@ -133,18 +133,10 @@
 
     static get observers() {
       return [
-        'filterChanged(filterType, entityId, startTime, endTime, cacheConfig, localize, language)',
+        'filterChangedDebouncer(filterType, entityId, startTime, endTime, cacheConfig, localize, language)',
       ];
     }
 
-    connectedCallback() {
-      super.connectedCallback();
```

Comment:
It is possible this element disconnected and then reconnected. We want to update on reconnect.

Reply:
I haven't encountered a situation where it would get reconnected and observers would not get called (either hassChanged or the multi-property observer)
There's probably no harm in putting it back with the debouncer in place. But what is the point in updating if neither hass nor the filters have changed anyway?

Reply:
Oh snap, sorry for missing this Andrey. We disconnect the more info when it's not visible.